mailboxes.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
This file is already present at: https://github.com/wireapp/wire-server/blob/develop/tools/api-simulations/mailboxes.example.json - perhaps you can update the description added above to point to that file instead?